documentation/docs/14-configuration.md

```diff
@@ -181,7 +195,7 @@ Enables or disables the client-side [router](#ssr-and-javascript-router) app-wid
 
 An object containing zero or more of the following values:
 
-- `exclude` - an array of glob patterns relative to `files.assets` dir. Files matching any of these would not be available in `$service-worker.files` e.g. if `files.assets` has value `static` then ['og-tags-images/**/*'] would match all files under `static/og-tags-images` dir.
+- `files` - a function with the type of `(filepath: string) => boolean`. When `true`, the following file will be available in `$service-worker.files`, otherwise it will be excluded.
```

Comment:
```suggestion
- `files` - a function with the type of `(filepath: string) => boolean`. When `true`, the given file will be available in `$service-worker.files`, otherwise it will be excluded.
```